easybuild/easyblocks/b/binutils.py

```diff
@@ -211,7 +219,7 @@ def sanity_check_step(self):
             ])
 
         # All binaries support --version, check that they can be run
-        custom_commands = ['%s --version' % os.path.join(self.installdir, 'bin', b) for b in binaries]
```

Comment:
The abs path was intentional, so those binaries are *really* used and not anything from the system. But I guess it doesn't matter due to PATH being set?

Reply:
Right before the sanity check the `binutils` module is loaded, so we're quite sure it's the correct binary.

The commands run in the sanity check should be run like an end user would.

We can extend the sanity check to verify the location of the binaries via the `which` function perhaps?